app/views/finance/npq/contracts/show.html.erb

```diff
@@ -0,0 +1,3 @@
+<% content_for :before_content, govuk_back_link(text: "Back", href: finance_npq_lead_provider_path(id: @npq_lead_provider.id)) %>
+
+<%= render Finance::NPQ::Contracts.new(contracts: @contracts) %>
```

Comment:
Could we rename this so that it clear at first sight with what type of instance we are dealing with without having to peruse through all the partials/controller? 

```suggestion
<%= render Finance::NPQ::Contracts.new(npq_contracts: @npq_contracts) %>
```